src/middlewared/middlewared/etc_files/pam.d/pam.inc.mako

```diff
@@ -3,9 +3,8 @@
         def __init__(self, **kwargs):
             self.middleware = kwargs.get('middleware')
             self.pam_mkhomedir = "pam_mkhomedir.so"
-            self.pam_ldap = "pam_ldap.so"
+            self.pam_ldap = "pam_sss.so"
```

Comment:
Hmm... it means a little more changes, but I think we should not use pam_ldap for pam_sss.